azurelinuxagent/common/osutil/default.py

```diff
@@ -322,7 +325,13 @@ def eject_dvd(self, chk_err=True):
         if chk_err and retcode != 0:
             raise OSUtilError("Failed to eject dvd: ret={0}".format(retcode))
 
-    def load_atappix_mod(self):
+    def try_load_atapiix_mod(self):
+        try:
+            self.load_atapiix_mod()
+        except Exception as e:
+            logger.warn("Could not load ATAPI driver: {0}".format(e.message))
```

Comment:
It works in Python 2, but you are right, this was deprecated in Python 3. Fixed - thanks for catching